### ISSUE TITLE
vue/v-on-function-call with "always" false negative 

### ISSUE THREAD

**Issue by Hyzual** [vuejs/eslint-plugin-vue#878]
[edit] It turns out this only happens for `vue-eslint-parser@^6.0.0`. The parsing changes cause this.

**Tell us about your environment**	
* **ESLint version:** 5.16.0	
* **eslint-plugin-vue version:** 5.2.2
* **vue-eslint-parser version:** 6.0.3
* **Node version:** v10.15.0

**Please show your full configuration:**
```js
module.exports = {
    plugins: ["vue"],
    extends: [
    ],
    parser: "vue-eslint-parser",
    rules: {
        "vue/v-on-function-call": ["error", "always"]
    }
};
```

**What did you do?**
I tried linting the following component in my project using the following command line:

```bash
npx eslint Component.vue
```

```vue
// Component.vue
<template><div @click="foo" v-on:mouseover="bar"></div></template>
```

**What did you expect to happen?**
I expected eslint to report an error

**What actually happened?**
```bash
$ npx eslint Component.vue
$ echo $?
0
```
No error reported.

**Comment by ota-meshi** [comment]
Thank you for this issue.

`eslint-plugin-vue@5` and `vue-eslint-parser@6` are not compatible. Please use `eslint-plugin-vue@6`.
